static/packages/App/Screenshots (View screenshots)/App/Screenshots_Viewer/config.json

```diff
@@ -0,0 +1,6 @@
+{
+"label":"Screenshots",
+"icon":"icon.png",
+"launch":"launch.sh",
+"description":"View screenshots"
```

Comment:
I would like it to be called label: "Gallery", description: "Screenshot viewer", and folder name in packages should be "Gallery (Screenshot viewer)".

Reply:
Fixed in [904982f](https://github.com/OnionUI/Onion/pull/115/commits/904982f375e1d2473668a44c2b9340e8569519e6)